src/language.js

```diff
@@ -571,6 +571,24 @@ const pages = {
       enableButton: 'Enable GFCR Indicators for this project',
       goToButton: 'Go to GFCR Indicators',
     },
+
+    citationHelperText:
+      'Citation to suggest for all uses of your project data. This citation will be displayed on the MERMAID dashboard and with any other data access method. ',
+    editCitation: 'Edit Citation',
+    editCitationModal: {
+      title: 'Edit Suggested Citation',
+      helper: 'Copy and paste project info into your citation',
+      projectName: 'Project Name',
+      projectAdmins: 'Project Admins',
+      otherProjectMembers: 'Other Project Members',
+      latestSampleEventDate: 'Latest Sample Event Date',
+      countries: 'Countries',
+      defaultCitation: 'Default Citation',
+      editCitation: 'Edit Citation',
+      cancel: 'Cancel',
+      updateCitation: 'Update Citation',
+      citationPreview: 'Citation Preview',
+    },
```

Comment:
We're cutting corners with translations, but I kinda felt like doing this for this one.